Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.0.1"
 edition = "2018"
 authors = ["github.com/containers"]
 description = "A container network stack"
+exclude = ["/.cirrus.yml", "/.github/dependabot.yml"]
```

Comment:
Can you exclude the full .github directory in case someone adds a new file in the future.

Reply:
Done, added hack dir as well.